torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2981,42 +3018,6 @@ TensorExprKernel::BackendType TensorExprKernel::inferBackendTypeFromDevice(
   return backendType;
 }
 
-static bool isValidIdentifierChar(char c, size_t pos) {
-  return islower(c) || isupper(c) || c == '_' || (pos > 0 && isdigit(c));
-}
-
-// replaces all invalid characters with underscore
-std::string sanitizeName(const std::string& input_name) {
-  std::stringstream sanitized_name;
-  for (size_t i = 0; i < input_name.size(); ++i) {
-    if (isValidIdentifierChar(input_name[i], i)) {
-      sanitized_name << input_name[i];
-    } else {
-      sanitized_name << "_";
-    }
-  }
-  return sanitized_name.str();
-}
-
-// we use the debug names in printing cuda code, they need to be removed
-// of characters that can't be used in a variable identifier
-void TensorExprKernel::genInputDebugNames() {
-  std::unordered_map<std::string, const torch::jit::Value*> name_to_value;
-  std::unordered_set<std::string> name_set;
-  std::unordered_map<const torch::jit::Value*, std::string> value_to_name;
-  for (const torch::jit::Value* input : graph_->inputs()) {
```

Comment:
can we just extend this to loop over constant nodes in the graph ? i would rather not modify the JIT IR, i think changing debug names and having them differ between the model graph and the optimized graph will hurt debugability